tsconfig.json

```diff
@@ -4,8 +4,8 @@
         "module": "es6",
         "declaration": true,
         "outDir": "./dist",
-        "strict": true
-
+        "strict": true,
+        "strictPropertyInitialization": false
```

Comment:
Could you please explain why?

Reply:
That is weird, 3 things you mention make the project didn't compile before. now it work. must be a monkey. I changed it back

Reply:
Sorry could you clarify what you did to build which resulted in the project not building?